tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
+   */
+  public List<Operation> getOperations() {
```

Comment:
Ah, I see. Yes, some iterator implementations do that but as far as I understand the [interface specification](https://docs.oracle.com/javase/7/docs/api/java/util/Iterator.html) does not require it (or even mention that they _could_ return this error).

So, yes, I agree that this iterator implementation shouldn't try to worry about throwing `ConcurrentModificationException`s. I'd even be okay with not talking about this detail in the javadoc at all - since it isn't part of the interface specification.

Thanks!